lib/dialects/sqlite/data-types.js

```diff
@@ -23,6 +23,17 @@ module.exports = BaseTypes => {
   BaseTypes.REAL.types.sqlite = ['REAL'];
   BaseTypes.DOUBLE.types.sqlite = ['DOUBLE PRECISION'];
   BaseTypes.GEOMETRY.types.sqlite = false;
+  BaseTypes.JSON.types.sqlite = ['JSON', 'JSONB'];
+
+  function JSONTYPE() {
```

Comment:
@sushantdhiman  Oh, I understood you want me to erase JSONB from line 26.

So, would I just leave these code as it is?